package.json

```diff
@@ -22,7 +22,7 @@
     "build": "wvr build --clean"
   },
   "dependencies": {
-    "@wvr/core": "2.2.4",
+    "@wvr/core": "git@github.com:TAMULib/Weaver-UI-Core.git#wwelling-patch-1",
```

Comment:
Please do not add the github URL dependency in the PR going into staging.
Noting that this PR requires a weaver update should be good enough.

Reply:
This is to afford testing of both Vireo and Weaver without additional effort from the reviewer.

Reply:
Please do not make this more difficult than it needs to be.